### PR TITLE
Add logging for Kong and Dynamo

### PR DESCRIPTION
**Pull request by marialivia16**

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -69,15 +69,21 @@ class Application(dynamo: DB, kong: Kong, val messagesApi: MessagesApi, val auth
...
-
-    Ok(views.html.editUser(id, filledForm, request.user.firstName, userKeys, editUserPageTitle))
+    dynamo.getUserWithId(id) match {
+      case Some(consumer) => {
+        val filledForm = editUserForm.fill(EditUserFormData(consumer.name, consumer.email, consumer.productName, consumer.productUrl, consumer.companyName, consumer.companyUrl))
+        val userKeys = dynamo.getKeysWithUserId(id)
+        Ok(views.html.editUser(id, filledForm, request.user.firstName, userKeys, editUserPageTitle))
+      }
+      case None => {
+        val userKeys = dynamo.getKeysWithUserId(id)
```
Comment (oilnam) [inline app/controllers/Application.scala]: We might want to move `val userKeys = dynamo.getKeysWithUserId(id)` outside the match since we use it twice?

Comment (marialivia16) [inline app/controllers/Application.scala]: @oilnam Yes, thanks! I didn't see that.

File: app/store/Dynamo.scala [hunk 2]
```diff
@@ -10,88 +10,178 @@ import play.api.Logger
...
+  }
+
+  def getUserWithId(id: String): Option[BonoboUser] = {
     val userQuery = new QuerySpec()
       .withKeyConditionExpression("id = :i")
       .withValueMap(new ValueMap().withString(":i", id))
       .withMaxResultSize(1)
-    BonoboTable.query(userQuery).asScala.toList.map(fromBonoboItem).head
+    val resultUser = BonoboTable.query(userQuery).asScala.toList.map(fromBonoboItem).headOption
+    Logger.info(s"DynamoDB: User with id $id is $resultUser")
```
Comment (oilnam) [inline app/store/Dynamo.scala]: Isn't logging "get" requests to the db a bit too much?

Comment (marialivia16) [inline app/store/Dynamo.scala]: @oilnam It is, I thought about it. Sometimes they are useful, but most of the time they are not. I think we can remove that log.

Comment (oilnam) [inline app/store/Dynamo.scala]: I'd remove all the "read" logs and only keep those that alter the state of the database (write/update).

File: app/controllers/Application.scala [hunk 3]
```diff
@@ -69,15 +69,20 @@ class Application(dynamo: DB, kong: Kong, val messagesApi: MessagesApi, val auth
   }
 
   def editUserPage(id: String) = maybeAuth { implicit request =>
-    val consumer = dynamo.getUserWithId(id)
-    val userKeys = dynamo.getAllKeysWithId(id)
-    val filledForm = editUserForm.fill(EditUserFormData(consumer.name, consumer.email, consumer.productName, consumer.productUrl, consumer.companyName, consumer.companyUrl))
-
-    Ok(views.html.editUser(id, filledForm, request.user.firstName, userKeys, editUserPageTitle))
+    val userKeys = dynamo.getKeysWithUserId(id)
+    dynamo.getUserWithId(id) match {
```
Comment (cb372) [inline app/controllers/Application.scala]: Ideally this kind of fix should be in a separate PR, as it has nothing to do with logging. 

It's fine to keep it in the PR this time, but it's good to get into the habit of keeping unrelated changes in separate PRs. It makes them easier to review, and it means that we can rollback one change (if it turns out to be buggy) without losing the other.

**Comment by oilnam** [comment]
:+1:
